lux/core/frame.py

```diff
@@ -181,12 +181,12 @@ def set_executor_type(self, exe):
                 import psycopg2
```

Comment:
Should we keep `set_executor_type` inside `frame.py` or can we move it all to config or somewhere else?

Reply:
Since the SQLconnection and executor parameters are now both in config, makes sense to move the set function for them there to.

Reply:
Ok let's move these set functions to config then.